### PR TITLE
generic_entity -> worker

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

File: src/worker.c [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        if (lastWorker == NULL) {
+            break;
+        }
+    }
+    ASSERT(i < MAX_WORKERS);
+
+    (*gCurrentWorkerListPtr)[i] = worker = heap_malloc(sizeof(*worker));
+    ASSERT(worker != NULL);
+
+    worker->flags = ENTITY_FLAGS_HIDDEN | ENTITY_FLAGS_DRAW_IF_CLOSE_HIDE_MODE1;
```
Comment (Unnunu) [inline src/worker.c:43]: Probably worker flags are different from entity flags

Comment (HailSanta) [reply src/worker.c]: Good catch.

**Comment by BowserSlug** [comment]
✅ This PR fixes 1 warnings!

**Review by ethteck — APPROVED** [review]

**Comment by ethteck** [comment]
great shtuff

**Pull request merged**